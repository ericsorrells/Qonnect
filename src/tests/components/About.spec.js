// ========================================================================================
import React from 'react';
import { shallow } from 'enzyme';
import 'jest-enzyme';
// ========================================================================================
import About from '../../components/About';
// ========================================================================================

describe('<About />', () => {
  const component = shallow(<About />);
  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });

  it('should include the correct text', () => {
    expect(component).toIncludeText('Coming Soon...');
  });
});