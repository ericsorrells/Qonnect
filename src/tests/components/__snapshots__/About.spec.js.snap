// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render correctly 1`] = `
<h1>
  Coming Soon...
</h1>
`;
